lib/nuget-parser/index.ts

```diff
@@ -36,12 +36,25 @@ function getPackagesFolder(packagesFolder, projectRootFolder) {
   return path.resolve(projectRootFolder, 'packages');
 }
 
+function getRootName(
+  root?: string,
```

Comment:
the first two params seem to not be tolerant to null, so why the optional typing?